File: fix-app-app-tsx.js
#!/usr/bin/env node;
;
import fs from 'fs';

console.log('🔧 Fixing app/App.tsx syntax errors...');

const content = `import React from 'react';
import { Helmet } from "react-helmet-async";

const AppPage = () => {
  return (
    
    <>
      <div>
    <Helmet />
          <title>App - Zion Tech Group</title>
          <meta name="description" content="App - Zion Tech Group" />
        </Helmet>
        <div>
    <div className="text-center"></div>
            <h1 className="text-4xl font-bold text-white mb-8">App</h1>
          </div>
        </div>}
      </div>})
    </>)}
  );}
};

export default AppPage;
`;

try {}
  fs.writeFileSync('/workspace/app/App.tsx', content, 'utf8');}
  console.log('✅ Fixed app/App.tsx');}
} catch (error) {}
  console.error('❌ Error fixing app/App.tsx: ', error.message);'}
}

File: fix_app_app_tsx.py
#!/usr/bin/env python3
"""
Script to fix the app/App.tsx file by converting broken import statements to proper React.lazy imports.
"""

import re

def fix_app_app_tsx():
    """Fix the app/App.tsx file by converting broken import statements to proper React.lazy imports."""
    
    with open('/workspace/app/App.tsx', 'r', encoding='utf-8') as f:
        content = f.read()
    
    # Pattern to match broken import statements
    pattern = r" import\('([^']+)'\)\);"
    
    def replace_import(match):
        path = match.group(1)
        # Extract the component name from the path
        if 'page' in path:
            component_name = path.split('/')[-2].replace('-', ' ').title().replace(' ', '') + 'Page'
        else:
            component_name = path.split('/')[-1].replace('.tsx', '').replace('-', ' ').title().replace(' ', '')
        return f"const {component_name} = lazy(() => import('{path}'));"
    
    # Replace all broken import statements
    new_content = re.sub(pattern, replace_import, content)
    
    with open('/workspace/app/App.tsx', 'w', encoding='utf-8') as f:
        f.write(new_content)
    
    print("Fixed app/App.tsx import statements")

if __name__ == "__main__":
    fix_app_app_tsx()